backend/src/test/java/com/staccato/memory/service/dto/request/MemoryReadRequestTest.java

```diff
@@ -0,0 +1,54 @@
+package com.staccato.memory.service.dto.request;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import com.staccato.memory.service.MemoryFilter;
+import com.staccato.memory.service.MemorySort;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MemoryReadRequestTest {
+    @DisplayName("필터가 주어졌을 때 올바른 MemoryFilter 리스트를 반환한다")
+    @Test
+    void getFiltersWithValidFilters() {
+        // given
+        MemoryReadRequest request = new MemoryReadRequest("TERM, term", "NEWEST");
+
+        // when
+        List<MemoryFilter> filters = request.getFilters();
+
+        // then
+        assertThat(filters).hasSize(1).containsOnly(MemoryFilter.TERM);
+    }
+
+    @DisplayName("필터가 주어지지 않았을 때 빈 리스트를 반환한다")
```

Comment:
이렇게 고치면 의미가 확실해져서 좋을 것 같습니다.

```suggestion
    @DisplayName("필터가 주어지지 않았을 때 빈 MemoryFilter 리스트를 반환한다")
```